.github/workflows/build-and-publish.yml

```diff
@@ -14,19 +14,43 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '10.17.0'
-          registry-url: https://npm.pkg.github.com/celonis
-
       - name: Yarn Install
         run: yarn install
 
       - name: Yarn Build
         run: yarn build
 
-      - name: Publish library
-        run: cd dist/ && npm publish --access public
+      - name: Move files to build
+        run: |
+          cp package.json dist/package.json
```

Comment:
Maybe we should do this using the build script instead. That way even if change the build workflows then it'll still work.
We already do this for `package.json`.

Reply:
I was thinking the same actually.

Reply:
I got a message that someone was trying to build the project on windows and it wasn't working because `cp` is not a command in cmd, that's why i moved these to a step in the build.